ocw/lib/EC2.py

```diff
@@ -76,6 +77,28 @@ def all_clusters(self):
             [clusters.append(cluster) for cluster in response['clusters']]
         return clusters
 
+    def delete_snapshots_older_than(self, delete_date: datetime.date):
```

Comment:
Make less complications in my brain for calling that function :)
```suggestion
    def delete_snapshots_older_than(self, age: datetime.timedelta):
```

Reply:
well it make complications in mine :) can you also highlight how than lines  https://github.com/cfconrad/pcw/pull/92/files#diff-c5779e6ba06607a1398b8f08e0bbbec9563650ba6688e2d3cc6569eddbc7e89eR87 and https://github.com/cfconrad/pcw/pull/92/files#diff-4b964610fda94cfeb813f5a21ed6975cc2ac100ac1546eb851ced96dc488af9bR55 will look like ?